docs/fake-model-api.md

```
@@ -0,0 +1,260 @@
+
+Run it from a throwaway directory unless you mean otherwise — `claude` runs in
+your current working directory and picks up that project's
+`.claude/settings.json`, hooks and permissions.
+
+Headless, one scripted answer:
+
+```sh
+scripts/claude-stub.py --text "hello from the stub" -- -p "say hello"
+```
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)